daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -568,6 +570,27 @@ createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not
         doesFileExist (cdaDir </> "ui" </> "build" </> "index.html") >>=
           assertBool "ui/build/index.html has been produced"
 
+        -- Now test that the messaging feature works by applying the necessary
+        -- changes and testing in the same way as above.
+        messagingPatch <- locateRunfiles (mainWorkspace </> "templates" </> "messaging-patch")
+        withCurrentDirectory cdaDir $ do
+          callCommandSilent $ "patch -s -p2 < " ++ messagingPatch
```

Comment:
It looks like on Linux this is not in the Bazel sandbox, you can probably get it from nix for now.

Reply:
Is there example of how to do this somewhere you could point me to? I tried adding to the WORKSPACE as with `tar` (https://github.com/digital-asset/daml/blob/master/WORKSPACE#L169) and to bazel.nix (https://github.com/digital-asset/daml/blob/master/nix/bazel.nix#L7) but I'm not sure what I'm doing here :)